tests/unit/common-helpers.spec.ts

```diff
@@ -629,4 +632,43 @@ describe("Common Helpers", () => {
       expect(isValidDate(new Date("2021-01-31T25:00:00.000Z"))).toBeFalsy();
     });
   });
+
+  describe("makeErrorFromAnything()", () => {
+    test.each([
+      [new Error("error"), "error"],
+      [
+        new ZodError([
+          {
+            code: "invalid_type",
+            expected: "string",
+            received: "number",
+            path: [""],
+            message: "invalid type",
+          },
+        ]),
+        `[\n  {\n    "code": "invalid_type",\n    "expected": "string",\n` +
+          `    "received": "number",\n    "path": [\n      ""\n` +
+          `    ],\n    "message": "invalid type"\n  }\n]`,
+      ],
+      [undefined, "undefined"],
+      [null, "null"],
+      ["string", "string"],
+      [123, "123"],
+      [{}, "[object Object]"],
+      [{ test: "object" }, "[object Object]"],
```

Comment:
Todo: I don't like it

Reply:
Actually, I think it's fine.
Just don't pass the bullshit to the `throw` statement :)